packages/contracts/contracts/plugin/PluginSetup.sol

```diff
@@ -21,18 +21,14 @@ abstract contract PluginSetup is ERC165, IPluginSetup {
     /// @inheritdoc IPluginSetup
     function prepareUpdate(
         address _dao,
-        address _plugin,
-        address[] memory _currentHelpers,
-        uint16[3] calldata _oldVersion,
-        bytes memory _data
+        uint16 _currentBuild,
```

Comment:
current seems better. Imagine you write `prepareUpdate`. The buildId that is passed is the build Id of the current state. It's also last, but current seems logical to me.

Reply:
Whichever feels more logical to you, I would always stick to it.

Either `current` everywhere or `last` everywhere. But not mixed because then it creates confusion